app/assets/stylesheets/modules/data_access.css.scss

```diff
@@ -192,31 +192,151 @@ section.data-access-next {
 
 .data-access-collections-list {
   list-style-type: disc;
-  padding-left: 2em;
+  margin-left: 2em;
+}
+
+.data-access-collections-list-panel {
+
+  list-style-type: none;
+  margin: 0;
+
+  .data-access-collection {
+    border: 1px solid #bbb;
+    border-left: 5px solid #34495e;
```

Comment:
This same color is used 4 times within this one file. Care to break this out into a variable? #letsgetsassy

Reply:
Makes sense. Will do.